<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>580</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 217, 217);
color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_abstract">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>451</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 102, 153);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_papertext">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>110</y>
     <width>391</width>
     <height>661</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 102, 153);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBut_to_main">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>707</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
background-color: rgb(221, 221, 221);</string>
   </property>
   <property name="text">
    <string>GO Back</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Users/88690/Downloads/arrow-left-circle (2).svg</normaloff>../../../../Users/88690/Downloads/arrow-left-circle (2).svg</iconset>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_ators">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>451</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 102, 153);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Img">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>580</y>
     <width>291</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>41</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 102, 153)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../Users/88690/Downloads/book-open (3).svg</pixmap>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_title">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>881</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 102, 153);
background-color: rgb(244, 244, 244);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
